### PR TITLE
Add homebrew to docs

### PR DESCRIPTION
**Pull request by jfo**
this also fixes a relative link in the download page header.

### CODE REVIEW

**Comment by jfo** [comment]
resolves #7

**Comment by andrewrk** [comment]
I think I'd like to add a section below called "Installing from a package manager" because we definitely don't have enough columns for 

 * Debian
 * Ubuntu
 * Homebrew
 * NixOS
 * Arch
 * Gentoo
 * Docker

File: www/download/index.html [hunk 1]
```diff
@@ -47,7 +45,29 @@ <h1>Releases</h1>
...
+        <td>
+          <a href="https://brew.sh/">Homebrew</a>
+        </td>
+        <td><a href="http://formulae.brew.sh/formula/zig"><pre>brew install zig</pre></a></td>
+      </tr>
+      <tr>
+        <td>
+          <a href="https://aur.archlinux.org/">AUR</a>
+        </td>
+        <td><a href="https://aur.archlinux.org/packages/zig/"><pre>pacman -S zig</pre></a></td>
```
Comment (jfo) [inline www/download/index.html]: Is this language correct? I haven't used Arch in a while.

File: www/download/index.html [hunk 2]
```diff
@@ -47,7 +45,29 @@ <h1>Releases</h1>
         <td><a href="http://ziglang.org/documentation/0.1.1">docs</a></td>
         <td><a href="https://s3.amazonaws.com/ziglang.org/download/0.1.1/zig-0.1.1.tar.xz">zig-0.1.1.tar.xz</a></td>
         <td><a href="https://s3.amazonaws.com/ziglang.org/download/0.1.1/zig-win64-0.1.1.zip">zig-win64-0.1.1.zip</a></td>
-        <td><pre>brew install zig</pre></td>
+      </tr>
+    </table>
+
+    <h1>Packages</h1>
```
Comment (jfo) [inline www/download/index.html]: Maybe this isn't accurate.

**Comment by jfo** [comment]
left out everything but AUR and homebrew, are any of the others merged as of now?

**Comment by jfo** [comment]
image

**Comment by jfo** [comment]
@andrewrk do you have a preference for either one of those layouts? Are the links and commands accurate? (mostly asking abt arch)

**Comment by tiehuis** [comment]
Arch is in the aur so that won't work. You could do `pacaur -S zig` or just provide a link to the repository, since there isn't a standard aur helper (pacaur is just one of the most common).

**Comment by tiehuis** [comment]
Thought about it a bit more, I'd just write `zig` in that field and let the user figure out what mechanism they want to use for downloading.

**Comment by jfo** [comment]
Thanks, how about this then:

image

Where the links go to the package pages.

**Comment by jfo** [comment]
we can revisit the layout if and when there are multiple releases/packages

**Comment by jfo** [comment]
image

**Comment by andrewrk** [comment]
apologies for the delay in getting to this. I'll try to merge this week at some point. where is that conversation where you guys were talking about markdown?

**Comment by jfo** [comment]
https://github.com/zig-lang/www.ziglang.org/issues/9

there is a way to have a minimal, offlineable, responsive-able site without mucking the gears up, markdown is ++

I'm [this](http://bettermotherfuckingwebsite.com/) not [that](http://motherfuckingwebsite.com/) these days, not style wise necessarily but just in the "it is possible to have nicely styled html with a minimal footprint"

No rush on this PR, we should add it to the main readme somewhere as well though.

**Comment by andrewrk** [comment]
merged. I put the content in a wiki page and linked to it.

**Pull request closed without merging**